### ISSUE TITLE
Products create price save wrong

### ISSUE THREAD

**Issue by lgastmans** [quentindecosne/sunlit#86]
When creating a new product the price is saved wrong
but the edit works fine

**Comment by quentindecosne** [comment]
I think it has to do with the acessor/mutator in Product.php

```php
    /**
     * Get the purchase price amount in decimal.
     *
     * @return string
     */
    public function getDisplayPurchasePriceAttribute()
    {
        $fmt = new NumberFormatter('en_IN', NumberFormatter::CURRENCY);
        
        return $fmt->format($this->purchase_price/100); //sprintf('%01.2f', $this->purchase_price / 100);
    }

    /**
     * Set the purchase price amount as integer.
     *
     * @param  string  $value
     * @return void
     */
    public function setPurchasePriceAttribute($value)
    {
        $this->attributes['purchase_price'] = $value * 100;
    }    
```

**Comment by quentindecosne** [comment]
Since you're x100 the price in the StoreProductRequest

```php

protected function prepareForValidation()
{

    $b = preg_replace( '/[^.\d]/', '', $this->display_purchase_price);
    if (!is_numeric($b) || (empty($b)))
        $b = 0;

    $this->merge([
        'purchase_price' => $b * 100,
    ]);

}
```

You don't need the SetPurchasePriceAttribute cause you're x100 twice
fixed with f8c0558067da9863b3993de8140b7f47db3b34df
